### DISCUSSION TITLE
Vercel does not show stack trace in 500 errors

### DISCUSSION THREAD

**Discussion by buddhabrot** [vercel/community#7011] [Help]
### Summary

I can't see a stack trace on my Function errors, which only happen on Vercel platform and not locally.

### Steps to Reproduce

I have a 500 on a function (a http api/... next.js api call)
I open View Logs on this Function.
I just see a http 500 line.
There doesn't seem to be a way to show an actual stack trace so I can debug the issue.
I do not have this issue locally, when running `vercel dev` everything works fine.

**Comment by QuiiBz** [comment]
Hi, could you share a minimal reproduction? This is definitely not intended, but I cannot reproduce on a brand new Next.js application.
